content/projects/linux/beginner/_index.md

```diff
@@ -81,3 +81,26 @@ Note: for each instruction write a script and save it under this task
 #### Resources
 
 1. https://www.lifewire.com/beginners-guide-to-nano-editor-3859002
+
+### Task 8 Update and Upgrade
+1. Update your system with `sudo apt update`. 
+
+2. Use the `sudo apt upgrade` to apply the updates downloaded and select Y for Yes. 
+
+When installing a package you use sudo apt install package_name.
+
+3. Install a specific package called tree;
+`sudo apt install tree`
+
+You might ask what is sudo? Sudo allows you to run programs with the security previledges of another user, it is also referred to as a superuser. It is often used to install, start and stop applications that require root priviledges.
+
+dpkg is a tool to intall, build remove and manage Debian Packages,
+This is how you use dpkg, we will install VS Code in this example.
+
+Firstly download VS Code [VS Code Download](https://code.visualstudio.com/download)
```

Comment:
Make this 4.

Reply:
If 3 works then, i'm assuming 4 will work

Reply:
We don't do assumptions here. Just look at it. With your eyes. hugo serve it and check for yourself.